### ISSUE TITLE
Add labels to stars (option)

### ISSUE THREAD

**Issue by Thiousi** [jenstornell/kirby-ratings#1]
It's often good practice to include a label for each star:
1: not happy
2: somewhat happy
etc..

This could be done in the blueprint I guess since you already list all the ratings!

**Comment by jenstornell** [comment]
But then I need to build a blueprint parser because blueprints are a part of the panel only. Maybe set it in the config instead?

**Comment by Thiousi** [comment]
There's a way to parse blueprints already. It's listed on your repo I believe
Oterhwise, use the language files not the config or you'll lock yourself in a position where you'll be annoyed when you make this multi-lingual

**Comment by jenstornell** [comment]
Alright. I'll think of a solution. The config now seems like the less good solution when you put it like that.

I like the language solution the best. The only trouble would probably be if people find it easier to use the blueprint.

I'll think about it. :)

**Comment by Thiousi** [comment]
Yeah the blueprint idea I didn't think it through... I saw that you listed the ratings already in there but that's on individual pages and it contains a number (the votes)
So it'd have to be somewhere else in the blueprint and you'd have to do a lot more guess work to figure out how to do it in your plugin...
